backend/openshift.deploy.yml

```diff
@@ -97,53 +99,62 @@ objects:
               name: ${NAME}
               env:
                 - name: NR_SPAR_BACKEND_VERSION
-                  value: ${NR_SPAR_BACKEND_VERSION}
+                  value: ${BUILD}
                 - name: ALLOWED_ORIGINS
-                  value: ${ALLOWED_ORIGINS}
+                  valueFrom:
+                    secretKeyRef:
+                      name: ${NAME}
+                      key: allowed_origins
                 - name: KEYCLOAK_REALM_URL
-                  value: ${KEYCLOAK_REALM_URL}
-                - name: POSTGRES_HOST
+                  valueFrom:
+                    secretKeyRef:
+                      name: ${NAME}
+                      key: keycloak-realm-url
+                - name: POSTGRESQL_HOST
                   value: ${NAME}-${ZONE}-database
-                - name: POSTGRES_DB
+                - name: POSTGRESQL_DATABASE
                   valueFrom:
                     secretKeyRef:
                       name: ${NAME}-${ZONE}-database
                       key: database-name
-                - name: POSTGRES_PASSWORD
+                - name: POSTGRESQL_PASSWORD
                   valueFrom:
                     secretKeyRef:
                       name: ${NAME}-${ZONE}-database
                       key: database-password
-                - name: POSTGRES_USER
+                - name: POSTGRESQL_USER
                   valueFrom:
                     secretKeyRef:
                       name: ${NAME}-${ZONE}-database
                       key: database-user
               ports:
-                - containerPort: 8090
+                - containerPort: 3000
                   protocol: TCP
               resources:
                 requests:
                   cpu: ${CPU_REQUEST}
                   memory: ${MEMORY_REQUEST}
+                limits:
+                  cpu: ${CPU_LIMIT}
+                  memory: ${MEMORY_LIMIT}
               readinessProbe:
                 httpGet:
-                  path: /check
-                  port: 8090
+                  path: /
+                  port: 3000
                   scheme: HTTP
-                initialDelaySeconds: 120
+                initialDelaySeconds: 60
                 periodSeconds: 15
-                timeoutSeconds: 10
+                timeoutSeconds: 5
               livenessProbe:
                 successThreshold: 1
                 failureThreshold: 3
                 httpGet:
-                  path: /check
-                  port: 8090
+                  path: /
```

Comment:
Keep an eye on these URLs. They are used by k8s to do readiness and liveness probes. if your application does not reply with a 200, it will kill the pod and redeploy.